README.md

```diff
@@ -61,7 +61,10 @@ It's 100% Open Source and licensed under the [APACHE2](LICENSE).
 ## Introduction
 
 This repo contains a GitHub Action that determines the affected [Atmos](https://atmos.tools) stacks for a PR, then
-triggers the corresponding spacelift stacks.
+creates a comment on the PR which Spacelift can use to trigger the corresponding stacks via a push policy.
+
+Optionally, you can use the `spacectl` trigger method, which uses the `spacectl` CLI to trigger the corresponding
+spacelift stacks directly rather than via comment/push policy.
 
 
```

Comment:
Would it make sense to explain when might one might want to use `spacectl` over using `comment` (vise versa)

Reply:
Possibly, but `spacectl` is probably not the preferred method except in the simplest of use cases. I really just left it there because I had already done the work for it and making the comment path an option and the default way wasn't too hard.